test/torchaudio_unittest/functional/batch_consistency_test.py

```diff
@@ -32,12 +32,10 @@ def assert_batch_consistency(self, functional, inputs, atol=1e-8, rtol=1e-5, see
         for i in range(1, len(inputs)):
             self.assertEqual(inputs[i].size(0), n)
         # Compute items separately, then batch the result
-        torch.random.manual_seed(seed)
         items_input = [[ele[i].clone() for ele in inputs] for i in range(n)]
         items_result = torch.stack([functional(*items_input[i]) for i in range(n)])
 
         # Batch the input and run
-        torch.random.manual_seed(seed)
```

Comment:
These two seeds cannot be removed. This helper function is supposed to run functions on the same seed twice and compare the results.